server/apps/immich/src/api-v1/album/album.controller.ts

```diff
@@ -72,6 +72,14 @@ export class AlbumController {
     return this.albumService.getAlbumInfo(authUser, albumId);
   }
 
+  @Get('/byAsset/:assetId')
```

Comment:
Hi @matthinc, I think you can make use of `GET /album?assetId=${id}` to list albums belong to that `assetId`.

Reply:
Good idea! It's indeed much cleaner to keep the number of api routes small. I'll refactor this later.